autotest/evaluate/eval_config_chat.py

```diff
@@ -0,0 +1,40 @@
+from mmengine.config import read_base
+from opencompass.models import OpenAISDK
+
+with read_base():
+    from opencompass.configs.datasets.gsm8k.gsm8k_gen_1d7fe4 import gsm8k_datasets  # noqa: F401, E501
+    from opencompass.configs.datasets.mmlu.mmlu_gen_4d595a import mmlu_datasets  # noqa: F401, E501
```

Comment:
cc @zhulinJulia24 can we change to the following dastaset:
```
*gpqa_datasets, *aime2025_datasets, *mmlu_pro_datasets
```

Reply:
After internal discussion, we'll leave those datasets to next version. The key point is how to involve and manage the judgers